src/classes/TDTM_DefaultConfig.cls

```diff
@@ -213,7 +213,22 @@ public without sharing class TDTM_DefaultConfig {
         handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
                 Class__c = 'ATTD_CourseConnectionContact_TDTM', Load_Order__c = 1, Object__c = 'Attendance_Event__c',
                 Owned_by_Namespace__c = 'hed', Trigger_Action__c = 'AfterInsert;AfterUpdate'));
+                
+        // Makes sure that the chosen Course Connection matches the chosen Contact and Course Offering
+        handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
+                Class__c = 'TG_ValidateData_TDTM', Load_Order__c = 1, Object__c = 'Term_Grade__c',
+                Owned_by_Namespace__c = 'hed', Trigger_Action__c = 'BeforeInsert;BeforeUpdate'));  
+
+        // Stops a Course Connection from being deleted if it has any Term Grade or Attendance Event
+        handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
```

Comment:
While I guess the system doesn't care, I would expect the Load Order for this class as well as the one below to have to be at least 2, since there are already classes with a Load Order of 1 for both of these objects.

Reply:
@robertbwatson, I've updated it to 2. Is this the best practice where if there's already classes with Load Order 1 for that same object, then it is best that for the new classes it should be 2 and if there's already 2, it should be 3? I'm still unfamiliar whether there's a good practice I need to follow.

Reply:
I don't really know if it's best practice, but it's probably not a bad idea. Where it really comes into play is if we have 2+ classes and it's critical that they run in a specific order.

(See https://powerofus.force.com/s/article/EDA-Manage-Trigger-Handlers and other TDTM documentation on the Hub)

Reply:
Hi there! The general rule would be that for those records where the order in which they run doesn't matter we give them all the same Load Order, starting with 1. Only where we actually care about load order we would increment the counter (if class X, Y, and Z have Load Order 1 and we need class W to run after Y, we would give W load order 2 - X and Z would keep order 1).

Reply:
Oh, thanks @ceiroa! @kdy618 in that case, it sounds like you were correct the first time and these can just remain at "1" instead.